docs/downloads.md

```diff
@@ -234,3 +242,31 @@ You'll need to extract the distribution-specific package, bundle it with necessa
 #### Symantec reports a virus `WS.Reputation.1` for the Windows builds
 
 This is a false positive reported because Symantec has not seen this file before -- see [this clarification](http://community.norton.com/forums/clarification-wsreputation1-detection) for details.
+
+#### How do I use it with [AWS Lambda](https://aws.amazon.com/lambda/) setups?
```

Comment:
Can you add it to the FaaS question and not mention a specific release? You should probably remove the reference for the linked ticket, as your information is more correct now.